baseplate/core.py

```diff
@@ -190,6 +190,19 @@ def configure_tracing(self, tracing_client, *args, **kwargs):
 
         self.register(TraceBaseplateObserver(tracing_client))
 
+    def configure_crash_reporting(self, client):  # pragma: nocover
```

Comment:
Fair enough. I just wanted to distinguish between handleable errors and fatal errors.